cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/ordered/AbstractTable.java

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright © 2014 Cask Data, Inc.
```

Comment:
I don't think IntelliJ updated this because it is a refactor/rename from AbstractOrderedTable, not really a new file.  Github is just showing it as a new file due to the amount of changes in the code.